### PR TITLE
Hub documentation

### PR DESCRIPTION
**Pull request by ldez**
### What does this PR do?

Add Hub documentation.

### Motivation

Add Hub documentation.

### More

- [ ] ~~Added/updated tests~~
- [x] Added/updated documentation

### Additional Notes

related to #8837

Co-authored-by: jbdoumenjou <jb.doumenjou@gmail.com>
Co-authored-by: Baptiste Mayelle <baptiste.mayelle@traefik.io>

### CODE REVIEW

File: docs/content/traefik-hub/index.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+This agent can:
+
+* get the Traefik metrics to display them in the Traefik Hub UI
+* secure the Traefik routers
+* provide ACME certificates to Traefik
+* transfer requests from the SaaS Platform to Traefik (and then avoid the users to expose directly their infrastructure on the internet)
+
+!!! warning "Traefik Hub Entrypoints"
+
+    When the Traefik Hub feature is enabled, Traefik exposes some services meant for the Traefik Hub Agent on a dedicated entrypoints (on ports `9900` and `9901` by default).
```
Comment (rtribotte) [inline docs/content/traefik-hub/index.md:17]:
```suggestion
    When the Traefik Hub feature is enabled, Traefik exposes some services meant for the Traefik Hub Agent on dedicated entrypoints (on ports `9900` and `9901` by default).
```

File: docs/content/traefik-hub/index.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    Given their sensitive nature, those services should not be publicly exposed.
+    Also this dedicated entrypoint, regardless of how it is created (default, or user-defined), should not be used by anything other than the Hub Agent.
```
Comment (rtribotte) [inline docs/content/traefik-hub/index.md:19]:
```suggestion
    Also those dedicated entrypoints, regardless of how they are created (default, or user-defined), should not be used by anything other than the Hub Agent.
```

**Review by kevinpollet — APPROVED** [review]
LGTM

**Review by rtribotte — APPROVED** [review]
LGTM 👌

**Pull request merged**